apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -513,6 +568,15 @@ const (
 	// - Work is not created/updated because of the unknown reasons.
 	ResourceWorkSynchronizedConditionType ResourcePlacementConditionType = "WorkSynchronized"
 
+	// ResourceOverriddenConditionType indicates whether all the selected resources have been overridden successfully
+	// before applying to the target cluster if there is any override defined.
+	// Its condition status can be one of the following:
+	// - "True" means all the selected resources are successfully overridden before applying to the target cluster or
+	// override is not needed if there is no override defined with the reason of NoOverrideSpecified.
+	// - "False" means some of them have failed.
+	// - "Unknown" means we haven't finished the override yet.
+	ResourceOverriddenConditionType ResourcePlacementConditionType = "ResourceOverridden"
+
```

Comment:
where is this condition supposed to apply to? We don't have a resourcePlacement yet.

Reply:
It's an new condition of the existing ResourcePlacementStatus https://github.com/Azure/fleet/blob/main/apis/placement/v1beta1/clusterresourceplacement_types.go#L444